docs/generators/typescript.md

```diff
@@ -24,6 +24,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |enumNameSuffix|Suffix that will be appended to all enum names.| |Enum|
 |enumPropertyNaming|Naming convention for enum properties: 'camelCase', 'PascalCase', 'snake_case', 'UPPERCASE', and 'original'| |PascalCase|
 |enumPropertyNamingReplaceSpecialChar|Set to true to replace '-' and '+' symbols with 'minus_' and 'plus_' in enum of type string| |false|
+|enumType|Specify the enum type which should be used in the client code.|<dl><dt>**stringUnion**</dt><dd>Union of literal string types</dd><dt>**enum**</dt><dd>Typescript's [string enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)</dd></dl>|stringUnion|
```

Comment:
please default to `enum` for backwards compatibility

Reply:
The rational here was that a breaking change was (kind of accidentally) introduced in https://github.com/OpenAPITools/openapi-generator/pull/14663.
So this pr fixes that.
Personally I would be ok with either solution.

Reply:
ah i see. well i guess i would still lean towards making `enum` the default, since 7.0.0 was released a while ago

Reply:
I've done it
Could you check it, please?